frontend/src/components/QuestionList.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Question from './Question';
+
+class QuestionList extends Component {
+  render() {
+    return (
+      <div>
+        {this.props.questions.map((question, i) => {
+          return <Question key={i} {...question} />;
+        })}
+      </div>
+    );
+  }
+}
+
+QuestionList.propTypes = {
+  /* Verify that questions follows the following schema
+    questions = [
+        {
+            title: ...
+            options: ...
+            correctAnswers ...
+        },
+        ...
+    ]
+    */
+  questions: PropTypes.arrayOf(propValue => {
```

Comment:
Maybe we can extract this PropType validator into its own function. Put it in a file that just contains validators / custom PropTypes?

Also, would be nice to define what the type of options (array of strings?) and correctAnswer (string?) is. 
e.g.
```
title: "What was the main character's name?",
options: ["Tom Sawyer", "Moby Dick", "Julius Caesar"],
correctAnswer: 1 // idk.. the index in options? or the string? I guess string lets us rearrange the order of options
```

Reply:
Not sure if we still want custom validators as I was a dummy and didn't know how to properly use prop-types but now the validator is much smaller and not really a function

Reply:
Yeah I think we don't need to extract it